cmd/client-s3.go

```diff
@@ -511,10 +511,8 @@ func (c *s3Client) Get(sseKey string) (io.Reader, *probe.Error) {
 	bucket, object := c.url2BucketAndObject()
 	var opts minio.GetObjectOptions
 	if sseKey != "" {
-		key := minio.NewSSEInfo([]byte(sseKey), "AES256")
-		for k, v := range key.GetSSEHeaders() {
-			opts.Set(k, v)
-		}
+		key := encrypt.DefaultPBKDF([]byte(sseKey), []byte(bucket+object))
```

Comment:
this is not correct @poornas DefaultPBKDF means we are taking password not sseKey as input. This would mean that 32byte long string is no longer a requirement. So you can simplify the code even further if you wish to use password-based style.  If not you should use `encrypt.NewSSEC` for now and we need to think about the UI again to support both key and password styles with @abperiasamy

Reply:
fixed. will discuss password-based style with @abperiasamy